getting_started/first_3d_game/06.jump_and_squash.rst

```diff
@@ -236,13 +236,13 @@ With this code, if no collisions occurred on a given frame, the loop won't run.
    func _physics_process(delta):
         #...
 
-        # Iterate through all collisions that occurred this frame
-        # in C this would be for(int i = 0; i < collisions.Count; i++)
+        # Iterate through all collisions and sliding movements that occurred this frame
+        # in C# this would be for(int i = 0; i < collisions.Count; i++)
         for index in range(get_slide_collision_count()):
             # We get one of the collisions with the player
             var collision = get_slide_collision(index)
 
-            # If the collision is with ground
+            # If the collision is null (it's a sliding movement)
```

Comment:
If the player collided is with ground, collision.get_collider() will return the ground (StaticBody3D), not null.